instrumentation/opentelemetry-instrumentation-falcon/src/opentelemetry/instrumentation/falcon/__init__.py

```diff
@@ -169,6 +178,15 @@ def __init__(self, *args, **kwargs):
         self._excluded_urls = get_excluded_urls("FALCON")
         super().__init__(*args, **kwargs)
 
+    def _handle_exception(
+        self, req, resp, ex, params
+    ):  # pylint: disable=C0103
+        # Falcon 3 does not execute middleware within the context of the exception
+        # so we capture the exception here and save it into the env dict
+        _, exc, _ = exc_info()
+        req.env[_ENVIRON_EXC] = exc
```

Comment:
I'm using the already defined but previously unused key `_ENVIRON_EXC` to store the status info.

Reply:
and this works for both 2 and 3?

Reply:
the `_handle_exception` also exists in Falcon 2, so it seems to work 😃